### PR TITLE
feat(prover): Add public object store to prover config

### PR DESCRIPTION
**Pull request by matias-gonz**
## What ❔
Add public object store to prover config

### CODE REVIEW

**Comment by Deniallugo** [comment]
You forgot to change from_env like here https://github.com/matter-labs/zksync-era/pull/2183/files

File: core/lib/env_config/src/fri_prover.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ impl FromEnv for FriProverConfig {
     fn from_env() -> anyhow::Result<Self> {
         let mut prover: FriProverConfig = envy_load("fri_prover", "FRI_PROVER_")?;
         prover.object_store = ObjectStoreConfig::from_env().ok();
+        prover.public_object_store = ObjectStoreConfig::from_env().ok();
```
Comment (Deniallugo) [inline core/lib/env_config/src/fri_prover.rs:9]: it should be `PublicObjectStore`

Comment (Deniallugo) [reply core/lib/env_config/src/fri_prover.rs]: please don't change `prover.object_store` it will be merged in a few minutes by another pr

**Comment by matias-gonz** [comment]
Closed in favor of #2187

**Pull request closed without merging**